web/src/main/webapp/css/classic/main.css

```diff
@@ -11,7 +11,6 @@
 
 BODY {
 	font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
-	font-size: 75%;
```

Comment:
I just had problems reading anything on a 15" display with 1080p resolution...

### 1440x960

![2016-03-09-082250_5760x1080_scrot](https://cloud.githubusercontent.com/assets/557430/13628249/e4746ac6-e5d0-11e5-8052-aa5788078b2b.png)

### 1024x768

![2016-03-09-082306_5760x1080_scrot](https://cloud.githubusercontent.com/assets/557430/13628248/e4731892-e5d0-11e5-8777-015bf515ef3d.png)


(:+1: for an additional, responsive interface overhaul!)